tests/baselines/reference/library_RegExpExecArraySlice.types

```diff
@@ -5,23 +5,23 @@ var regExpExecArrayValue: RegExpExecArray;
 >RegExpExecArray : RegExpExecArray
 
 regExpExecArrayValue.slice();
->regExpExecArrayValue.slice() : string[]
->regExpExecArrayValue.slice : (start?: number, end?: number) => string[]
+>regExpExecArrayValue.slice() : RegExpExecArray
+>regExpExecArrayValue.slice : (start?: number, end?: number) => RegExpExecArray

```

Comment:
Wrong, must return `string[]`. It can avoid by overloading in RegExpExecArray and RegExpMatchArray.